### PR TITLE
add instructions for how to sync docker images with Satellite

### PR DESCRIPTION
**Pull request by opuk**
#### What is this PR About?
Adding information on how to sync required images for OCP using Satellite.

#### How should we test or review this PR?
The best way is probably to use the script in https://github.com/redhat-cop/openshift-toolkit/pull/21
and deploying a OCP cluster with the variables described.

#### Is there a relevant Trello card or Github issue open for this?
None that I could find.

#### Who would you like to review this?
No preference.

### CODE REVIEW

File: playbooks/installation/index.adoc [hunk 1]
```diff
@@ -592,7 +592,25 @@ TODO
...
+There is a script available at https://github.com/redhat-cop/openshift-toolkit/tree/master/satellite/populate-ocp.sh that can be used to save time.
+
+By adding the below 2 lines in your inventory file you configure OpenShift to consume the images from Satellite that was synced above.
+
+----
+oreg_url=satellite.example.com:5000/lab-ocp_docker_images-openshift3_ose-${component}:${version}
+cli_docker_additional_registries=satellite.example.com:5000
+----
+
+Satellite does not give you an option of the above naming scheme so you will need to modify imagestreams accordingly.
```
Comment (etsauer) [inline playbooks/installation/index.adoc]: @opuk can you expand and explain here how that naming scheme is constructed? For example, I can kind of figure out that `ocp_docker_images` comes from the product name, but where does the `lab-` prefix come from? etc.,

File: playbooks/installation/index.adoc [hunk 2]
```diff
@@ -592,7 +592,25 @@ TODO
...
+
+----
+hammer product create --name "OCP Docker Images" --organization "c1-ocp.myorg.com"
+hammer repository create --name "openshift3/ose" --organization-id "c1-ocp.myorg.com" --content-type docker --url "https://registry.access.redhat.com" --docker-upstream-name "openshift3/ose" --product "OCP Docker Images"
+----
+
+NOTE: Satellite will sync ALL tags for each image repository you create so it might be space intensive.
+
+There is a script available at https://github.com/redhat-cop/openshift-toolkit/tree/master/satellite/populate-ocp.sh that can be used to save time.
+
```
Comment (etsauer) [inline playbooks/installation/index.adoc]: can you include a sample command to run the script? just for completion.

**Comment by opuk** [comment]
@etsauer I hope that explains things better. Let me know if you want me to squash the commits.

**Comment by etsauer** [comment]
@opuk looks good on the naming scheme. still two other items (see above comments) i'd like to see:

* update http://playbooks-rhtconsulting.rhcloud.com/playbooks/installation/#container-image-content to remove `(Coming Soon!) and instead link to this new section
* A sample command showing the `populate-ocp.sh` script usage.

**Comment by etsauer** [comment]
@opuk bump.

**Comment by opuk** [comment]
@etsauer sorry about the delay, missed a broken link

File: playbooks/installation/index.adoc [hunk 3]
```diff
@@ -311,7 +311,7 @@ This proposed architecture installs OpenShift via RPM. This is the most common w
 In addition to RPM content, OpenShift requires the ability to pull container images from an external image registry. In order to bring this in house we suggest the following options to build and sync a Standalone Registry.
 
 * A Simple Bootstrap Registry (link:#simple-docker-registry-docker-distribution[See below.])
-* Using the OpenShift Standalone Registry (Coming soon)
+* Using the OpenShift Standalone Registry (link:#syncing-images-using-satellite-6[See below])
```
Comment (jtudelag) [inline playbooks/installation/index.adoc]: Shouldn't it be this instead?
Syncing image content to Satellite 6 (link:#syncing-images-using-satellite-6[See below])

File: playbooks/installation/index.adoc [hunk 4]
```diff
@@ -592,7 +592,35 @@ TODO
...
+
+----
+$ ./populate-docker.sh
+----
+
+By adding the below 2 lines in your inventory file you configure OpenShift to consume the images from Satellite that was synced above.
+
+----
+oreg_url=satellite.example.com:5000/lab-ocp_docker_images-openshift3_ose-${component}:${version}
+cli_docker_additional_registries=satellite.example.com:5000
```
Comment (jtudelag) [inline playbooks/installation/index.adoc]: Is cli_docker_additional_registries variable still valid? I dont see it in anywhere in the openshift_ansible repo...

Comment (opuk) [reply playbooks/installation/index.adoc]: Ah, no should be openshift_ now, I fix.

File: playbooks/installation/index.adoc (hunk 3 above)
Comment (opuk) [reply playbooks/installation/index.adoc]: Yes, good catch.

**Pull request merged**